### DISCUSSION TITLE
Feat.req. Win builds in Releases tab

### DISCUSSION THREAD

**Discussion by necros2k7** [ImageMagick/ImageMagick#3871] [General]
subj

**Comment by dlemstra** [comment]
I have no idea what you are asking us.

**Reply by necros2k7** [reply]
> I have no idea what you are asking us.
There are no windows builds only appimage
![no win buld](https://user-images.githubusercontent.com/16452453/123516615-1f912380-d6f1-11eb-9530-a34ce457cdfa.JPG)

**Reply by dlemstra** [reply]
The releases for Window are created in a separate process. They can be found here instead: https://imagemagick.org/script/download.php#windows

**Comment by necros2k7** [comment]
What I ask is to release Win builds here)

**Reply by dlemstra** [reply]
I know what you are asking. But we will not add them to the release here. You can download them from our site.
